### PR TITLE
Add documentation for Unifi LED integration

### PR DESCRIPTION
**Pull request by florisvdk**
**Description:**
Added unifiled integration documentation in preparation of making a pull request for it.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#27475

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/unifiled.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+- [Light](#light)
+
+## Configuration
+
+This integration uses settings in the configuration file. Underneath here is a example.
+
+```yaml
+# Example configuration.yaml entry
+light:
```
Comment (klaasnicolaas) [inline source/_integrations/unifiled.markdown:21]: Only required items in de first example

File: source/_integrations/unifiled.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - platform: unifiled
+    host: 192.168.1.20
```
Comment (klaasnicolaas) [inline source/_integrations/unifiled.markdown]:
```suggestion
    host: IP_ADDRESS
```

File: source/_integrations/unifiled.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+{% configuration %}
+host:
+  description: Ip address or DNS name used to connect to the Unifi LED controller.
+  type: string
+  required: true
+  default: None
+port:
+  description: Port used to connect to the Unifi LED controller.
+  type: integer
```
Comment (klaasnicolaas) [inline source/_integrations/unifiled.markdown]: In the code it is defined as string

File: source/_integrations/unifiled.markdown [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  required: true
```
Comment (klaasnicolaas) [inline source/_integrations/unifiled.markdown]:
```suggestion
  required: false
```

File: source/_integrations/unifiled.markdown [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+# Example configuration.yaml entry
+light:
+  - platform: unifiled
+    host: 192.168.1.20
+    port: 20443
+    username: ubnt
+    password: ubnt
+```
+
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_integrations/unifiled.markdown:28]: Is there also the option ID? [See here](https://github.com/home-assistant/home-assistant/pull/27475/files#diff-6ff9207b02057a414a71a1e032da1acaR31)

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
I left som comments, btw please rebase this PR to the `next` branch.

File: source/_integrations/unifiled.markdown (hunk 5 above)
Comment (florisvdk) [reply source/_integrations/unifiled.markdown:28]: That is something left over from testing, removing it from the integration.

**Comment by florisvdk** [comment]
I think i messed something up with the rebasing. I am new to a lot of this.

Is this fixable or do i make a new pull request?

**Comment by klaasnicolaas** [comment]
Now you only chaged the target branch, but you must also rebase the `patch-2` branch of your fork.

**Comment by florisvdk** [comment]
Finally figured it out, Thanks for putting up with this.

File: source/_integrations/unifiled.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_integrations/unifiled.markdown:21]: On codebase I see username and password as optional not required 😉

File: source/_integrations/unifiled.markdown [hunk 6]
```diff
@@ -0,0 +1,55 @@
+---
+title: "UniFi LED"
+description: "Instructions on how to configure the UniFi LED integration with UniFi LED Controller by Ubiquiti."
+logo: ubiquiti.png
+ha_category:
+  - Light
+ha_release: next
```
Comment (klaasnicolaas) [inline source/_integrations/unifiled.markdown]:
```suggestion
ha_release: 0.101
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by klaasnicolaas** [comment]
Good to see that you have succeeded in solving the problem, with a few small comments left we can approve it 😉

**Comment by florisvdk** [comment]
Made a new commit to fix the settings in the integration code.

**Comment by florisvdk** [comment]
@klaasnicolaas I fixed the things you asked for.

**Review by klaasnicolaas — DISMISSED** [review]

File: source/_integrations/unifiled.markdown [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+password:
+  description: Password used to log into the Unifi LED controller.
+  type: string
+  required: true
+  default: None
+{% endconfiguration %}
+
+## Light
+
+The light panels output state and brightness are synchronised with home assistant.
```
Comment (rytilahti) [inline source/_integrations/unifiled.markdown]: What does this section deliver to the reader?

File: source/_integrations/unifiled.markdown [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+```
+
+{% configuration %}
+host:
+  description: Ip address or DNS name used to connect to the Unifi LED controller.
+  type: string
+  required: true
+  default: None
+port:
+  description: Port used to connect to the Unifi LED controller.
```
Comment (rytilahti) [inline source/_integrations/unifiled.markdown:35]: Is it necessary to make this configurable at all? Other than that, description such as `Control Port`, `Login username` would be shorter and easier to read.

File: source/_integrations/unifiled.markdown [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+light:
+  - platform: unifiled
+    host: IP_ADDRESS
+    username: USERNAME
+    password: PASSWORD
+```
+
+{% configuration %}
+host:
+  description: Ip address or DNS name used to connect to the Unifi LED controller.
```
Comment (rytilahti) [inline source/_integrations/unifiled.markdown]: Maybe `IP address or hostname`?

File: source/_integrations/unifiled.markdown [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+[UniFi LED](https://unifi-led.ui.com/) by [Ubiquiti Networks, inc.](https://www.ubnt.com/) is a system off controler managed led light panels and dimmers.
+
+There is currently support for the following device type within Home Assistant:
+
+- [Light](#light)
+
+## Configuration
+
+This integration uses settings in the configuration file. Underneath here is a example.
```
Comment (rytilahti) [inline source/_integrations/unifiled.markdown]: Unnecessary, IMO.

File: source/_integrations/unifiled.markdown [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+logo: ubiquiti.png
+ha_category:
+  - Light
+ha_release: 0.101
+ha_iot_class: Local Polling
+---
+
+[UniFi LED](https://unifi-led.ui.com/) by [Ubiquiti Networks, inc.](https://www.ubnt.com/) is a system off controler managed led light panels and dimmers.
+
+There is currently support for the following device type within Home Assistant:
```
Comment (rytilahti) [inline source/_integrations/unifiled.markdown:13]: Unnecessary, IMO.

File: source/_integrations/unifiled.markdown [hunk 12]
```diff
@@ -0,0 +1,55 @@
...
+title: "UniFi LED"
+description: "Instructions on how to configure the UniFi LED integration with UniFi LED Controller by Ubiquiti."
+logo: ubiquiti.png
+ha_category:
+  - Light
+ha_release: 0.101
+ha_iot_class: Local Polling
+---
+
+[UniFi LED](https://unifi-led.ui.com/) by [Ubiquiti Networks, inc.](https://www.ubnt.com/) is a system off controler managed led light panels and dimmers.
```
Comment (rytilahti) [inline source/_integrations/unifiled.markdown]: This needs rewriting.

File: source/_integrations/unifiled.markdown (hunk 7 above)
Comment (florisvdk) [reply source/_integrations/unifiled.markdown]: It gives the reader information about which data is transfered over from the controller.

**Review by frenck — APPROVED** [review]
Thanks, @florisvdk! 👍 Sorry for the long wait! 🙏

**Pull request merged**